cherrypy/_cpcompat.py

```diff
@@ -332,3 +332,13 @@ def _args_from_interpreter_flags():
             args.append('-W' + opt)
 
         return args
+
+if six.PY2:
+    from cgi import escape as escape_html
+else:
+    from html import escape
+
+    def escape_html(s):
```

Comment:
You break function signature here. It's now impossible to pass custom `quote` argument explicitly. You should accept  `quote=False` and bypass it on call below.